contents/blog/best-open-source-analytics-tools.md

```diff
@@ -11,221 +11,301 @@ featuredImageType: full
 categories: ["Guides", "Open source"]
 ---
 
-Most well-known analytics tools demand hefty price tags, and they own your data. This is bad news for early-stage startups, or anyone who cares about user privacy. Thankfully, open-source analytics tools are better, and more numerous, than ever before.
+It's a truth universally acknowledged that every business reliant on a monolithic enterprise product has need of an open source alternative... probably.
 
-Most open source alternatives to the likes of Google Analytics, Amplitude, or Mixpanel are either free, or significantly cheaper. They also allow you to modify and extend them however you wish.
+In this guide we're looking at open-source analytics tools, which we've split into two broad categories:
 
-Here, we’ve selected some of the best open-source analytics tools across a range of categories, including big data platforms, specialized web analytics tools, A/B testing tools, [open-source feature flag tools](/blog/best-open-source-feature-flag-tools), and all-in-one platforms like PostHog.
+- [Open-source analytics tools](#the-best-open-source-analytics-tools) that collect data and help you understand user behavior. This includes product analytics platforms, but also web analytics, session recording, and AB testing tools.
 
-## 1. [PostHog](https://posthog.com/)
+- [Open-source data analytics](#the-best-open-source-data-and-bi-tools) and business intelligence tools that connect to data sources for building dashboards, insights, interactive visualizations, and real-time data logging.
+
+We've chosen each entry carefully based on a broad range factors, such as popularity and user growth, community size and activity, our own experience using them, and publicly available feedback and sentiment.
+
+We have separate guides on [open-source feature flag tools](/blog/best-open-source-feature-flag-tools) and [open-source AB testing products](/blog/best-open-source-ab-testing-tools).
+
+> **Changelog** 
+> - Jun 16, 2022: Added Fathom Lite, Open Web Analytics, GrowthBook, OpenReplay
+> - Aug 24, 2022: Removed Fathom Lite, Open Web Analytics; added Apache Superset, Redash
+
+## The best open source analytics tools
+
+This section is dedicated to tools that actively collect data on user behavior. These range from the very simple, such as the privacy-focused Plausible, to broad analytics tools like Matomo.
+
+### PostHog
 ![PostHog - best open source analytics tools](../images/blog/gdpr-compliant-analytics/posthog-gdpr-compliant.png)
 
-PostHog is an all-in-one open-source analytics platform that offers all the essential tools you need to build better products, including funnels, feature flags, path analysis, A/B testing, heat maps, and more – it's also one of the [best open source session recording tools](/blog/best-open-source-session-replay-tools) around. 
+- **GitHub Stars:** 8.9k
```

Comment:
```suggestion
- **GitHub Stars:** 8.9k - [Add a star!](https://github.com/PostHog/posthog)
```

Reply:
Not going to add this as it feels a bit much. Trying to avoid too much overt self-promotion